1Hive/celeste-dashboard
# [Fix] The localhost:8545 error in dashboard

Resolve https://github.com/1Hive/celeste-dashboard/issues/154

That PR make changes to check if  getDefaultProvider(null) is received, threat like getDefaultProvider(undefined) using the default values.

![image](https://user-images.githubusercontent.com/691510/155822009-3b335131-32ee-4803-8486-6e37fff52515.png)

Too added jest files

![image](https://user-images.githubusercontent.com/691510/155822026-4542e6e8-34c0-4fd3-9e50-140a0227f005.png)

## Another modifications

To be able to build that isseu the BundleWatch was disabled and 3box removed.

## How to test?

1 - Open https://celeste-dashboard-dk1sam5d7-1hive.vercel.app/
2 - Check Dev Tools Console
